boards/arm/stm32wb5mm_dk/doc/stm32wb5mm_dk.rst

```diff
@@ -130,13 +134,15 @@ Bluetooth and compatibility with STM32WB Copro Wireless Binaries
 To operate bluetooth on STM32WB5MMG, Cortex-M0 core should be flashed with
 a valid STM32WB Coprocessor binaries (either 'Full stack' or 'HCI Layer').
 These binaries are delivered in STM32WB Cube packages, under
-Projects/STM32WB_Copro_Wireless_Binaries/STM32WB5x/
+``Projects/STM32WB_Copro_Wireless_Binaries/STM32WB5x/``.
+
 For compatibility information with the various versions of these binaries,
 please check `modules/hal/stm32/lib/stm32wb/hci/README`_
-in the hal_stm32 repo.
-Note that since STM32WB Cube package V1.13.2, "full stack" binaries are not compatible
-anymore for a use in Zephyr and only "HCI Only" versions should be used on the M0
-side.
+in the ``hal_stm32`` repo.
+
+Note that since STM32WB Cube package V1.13.2, `"full stack"` binaries are not
+compatible anymore for a use in Zephyr and only `"HCI Only"` versions should be
```

Comment:
```suggestion
Note that since v1.13.2 of STM32WBCube package, `"full stack"` binaries are no
longer compatible for use in Zephyr and only `"HCI Only"` versions should be
```